lib/types.tsx

```diff
@@ -205,3 +205,21 @@ export interface CrudOrganisationResponse {
     ];
   };
 }
+
+// File Upload Result Type
+export type UploadSuccess = {
+  isValid: true;
+  successValue: SuccessData;
+};
+
+export type SuccessData = {
+  location: string;
+  data?: string;
+};
+
+export type UploadFailure = {
+  isValid: false;
```

Comment:
When defining a type or an interface it's best practice to avoid using actual values.
For example `isValue: true` would be usually typed as `isValid: boolean`

Reply:
Will address this one in a coming PR.